src/index.js

```diff
@@ -31,66 +31,68 @@ const detectNewETHAddress = async (str) => {
-    setAsync('ogMembers', JSON.stringify(Array.from(ogMembers))),
-    setAsync(address, user.id),
-    setAsync(`${user.id}|address`, address),
-  ]);
-};
+// const registerAddress = async (user, address) => {
+//   const ogMembers = new Set(JSON.parse((await getAsync('ogMembers')) || '[]'));
+//   ogMembers.add(address);
+//   await Promise.all([
+//     setAsync('ogMembers', JSON.stringify(Array.from(ogMembers))),
+//     setAsync(address, user.id),
+//     setAsync(`${user.id}|address`, address),
+//   ]);
+// };
 
 class DiscordActions {
   constructor(client) {
     this.client = client;
   }
 
-  async genericMessage(message) {
-    const { author, channel, content } = message;
-    const isDM = channel.constructor === DMChannel;
-    const dmChannel = isDM ? channel : (await author.createDM());
+   async genericMessage(message) {
+     const { author, channel, content } = message;
+     const isDM = channel.constructor === DMChannel;
+     const dmChannel = isDM ? channel : (await author.createDM());
     console.log('Got a generic message', content, 'on channel', channel.id, 'from user', author);
-
-    if (content.match(/^!jelly_init/)) {
-      message.delete();
-      this.guildMemberJoin(message);
-      return;
-    }
-
-    if (content.match(/^!jelly_forget_me/)) {
-      const storedAddress = await getAsync(`${author.id}|address`);
-      deleteAddress(author, storedAddress);
-      message.delete();
-      dmChannel.send('I no longer know what your address is. If you want me to have it, message me with it again.');
-      return;
-    }
-
-    if (content.match(/^!jelly_jam/)) {
-      new JellyJamGame(message.channel);
-      message.delete();
-      return;
-    }
-
-    if (content.match(/^reset_jelly_jam/)) {
-      jellyJamRound = 0;
-      message.delete();
-      return;
-    }
-
-    const address = await detectNewETHAddress(content);
-
-    if (address && isDM) {
-      const storedAddress = await getAsync(`${author.id}|address`);
-      if (storedAddress) {
-        await deleteAddress(author, storedAddress);
-      }
-
-      await registerAddress(author, address);
+   }
+  
+
+    //  if (content.match(/^!jelly_init/)) {
+    //   message.delete();
+    //   this.guildMemberJoin(message);
+    //   return;
+    // }
+
+  //   if (content.match(/^!jelly_forget_me/)) {
+  //     const storedAddress = await getAsync(`${author.id}|address`);
+  //     deleteAddress(author, storedAddress);
+  //     message.delete();
+  //     dmChannel.send('I no longer know what your address is. If you want me to have it, message me with it again.');
+  //     return;
+  //   }
+
+  //   if (content.match(/^!jelly_jam/)) {
+  //     new JellyJamGame(message.channel);
+  //     message.delete();
+  //     return;
+  //   }
+
+  //   if (content.match(/^reset_jelly_jam/)) {
+  //     jellyJamRound = 0;
+  //     message.delete();
+  //     return;
+  //   }
+
+  //   const address = await detectNewETHAddress(content);
```

Comment:
leave this in place (through line 95) so we can still ask them for an address in the future